code-bar-job-board-app/src/components/App/App.css

```diff
@@ -9,6 +9,11 @@ body {
     font-weight: bold;
 }
 
+.button {
+    margin-right: 10px;
+    margin-bottom: 10px;
+}
+
```

Comment:
We can avoid this type of styling by using the bootstrap margin classes here https://getbootstrap.com/docs/5.1/utilities/spacing/#margin-and-padding